### ISSUE TITLE
Remove `recurrance` field from the `Event` schema.

### ISSUE THREAD

**Issue by meetulr** [PalisadoesFoundation/talawa-api#2107]
### Describe the bug

A field `recurrance` was added to the `Event` schema to store recurrence frequency ("Daily" / "Monthly" etc.) while implementing recurring events.
```javasctipt
    recurrance: {
      type: String,
      required: function (this: InterfaceEvent): boolean {
        return this.recurring;
      },
      enum: ["ONCE", "DAILY", "WEEKLY", "MONTHLY", "YEARLY"],
      default: "ONCE",
    },
```
```javascript
  type Event {
    ...
    recurrance: Recurrance
  }
```
```javascript
  enum Recurrance {
    DAILY
    WEEKLY
    MONTHLY
    YEARLY
    ONCE
  }
```

The approach has been changed since then, and this field is no longer required. The recurrence specific properties are accessed through the new `recurrenceRule` field on the `Event` schema, which was added recently with this PR #2095.
We need to remove every usage & reference of the old `recurrance` field, to eliminate possible confusion.

### Solution

1) Remove the `recurrance` field from the `Event` mongoose schema.
2) Remove the `recurrance` field from the `Event` graphql schema.
3) Remove all references & uses of `recurrance` fields in all the types, mutations, queries, tests, i.e. from anywhere.

**Comment by palisadoes** [comment]
Does this change your documentation in Talawa-Docs?

**Comment by meetulr** [comment]
> Does this change your documentation in Talawa-Docs?

1) No, this field was being used in the previous approach, before I took on the recurring events issue. I planned to remove it separately after the CRUD functionalities were implemented, because this field was used in many files, and could have caused confusion in the CRUD PRs.

2) Though the docs would need to include the new recurrence properties I've added in my previous PR#2095.
